lib/features/composer/presentation/model/font_name_type.dart

```diff
@@ -0,0 +1,28 @@
+
+enum FontNameType {
+  sansSerif,
+  times,
+  courier;
```

Comment:
three fonts family are supported by the library? only three? if we want to support more, how to add?

Reply:
- Simply add `fontFamily` that web supported. If you want to add some other font you have to include a link in the `head` tag of the html for it. Ex:
```html
<head>
<link href='https://fonts.googleapis.com/css?family=Sofia' rel='stylesheet'>
<style>
body {
     font-family: 'Sofia';font-size: 22px;
}
</style>
</head>
```
You can read more here [https://www.w3schools.com/cssref/css_websafe_fonts.asp](https://www.w3schools.com/cssref/css_websafe_fonts.asp)